compiler/noirc_evaluator/src/errors.rs

```diff
@@ -55,7 +55,7 @@ pub enum RuntimeError {
     #[error("Could not resolve some references to the array. All references must be resolved at compile time")]
     UnknownReference { call_stack: CallStack },
     #[error("{invalid_input_bit_size}")]
-    InvalidInputBitSize { invalid_input_bit_size: InvalidInputBitSize, call_stack: CallStack },
```

Comment:
Apologies when I first reviewed I didn't notice we were using `acir:: InvalidInputBitSize` inside these evaluator errors. I don't think we should do that to avoid propagating the field through `RuntimeError`. I was looking at OpcodeResolutionError where we already have a generic field as part of its definition. In the evaluator we can have an error similar to `UnsupportedIntegerSize`.